### PR TITLE
Updated project home; rerendered with smithy

### PR DESCRIPTION
**Pull request by dschreij**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dschreij** [comment]
@conda-forge/core. The package builds correctly on the CIs, but they all fail because they can't find the LICENSE.txt file in the recipe root dir. This file is present in the repo; why can't they find it? Am I overlooking something?

**Comment by dschreij** [comment]
Apologies @conda-forge/core, I see that the license file now also has to be included in the recipe folder, and not the repo root.
The new problem is that on some CIs the module fails to be found during import, while on others it all works flawlessly. I'm not sure how to proceed now.

**Comment by ocefpaf** [comment]
> Apologies @conda-forge/core, I see that the license file now also has to be included in the recipe folder, and not the repo root.

It has always been in the source tarball first and, if missing, in the recipe directory.

> The new problem is that on some CIs the module fails to be found during import, while on others it all works flawlessly. I'm not sure how to proceed now.

That is odd. Can you see anything out of order in the log during the build phase?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ test:
 about:
   home: https://github.com/marcusva/py-sdl2
   license: Public Domain/ZLIB
-  license_file: LICENSE.txt
+  license_file: doc/copying.rst
```
Comment (ocefpaf) [inline recipe/meta.yaml:41]: 👍

**Comment by dschreij** [comment]
> It has always been in the source tarball first and, if missing, in the recipe directory.

You're right! That was a lapse on my behalf. They moved the license file in the source, so that's why it couldn't be found. 
There are no problems during the build/install, and this error only occurs during the testing phase. The weird thing is that the pysdl2 module is also correctly listed in the manifest:

```
pysdl2:          0.9.6-1              local      
python:          3.7.3-h5b0a415_0     conda-forge
...
sdl2:            2.0.9-he1b5a44_0     conda-forge
```

**Comment by dschreij** [comment]
Builds seem to fail for any version of python but 3.7. There is a warning about conda-verify, but that doesn't seem harmful:

```
Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
WARNING:conda_build.build:Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
TEST START: /usr/local/miniconda/conda-bld/osx-64/pysdl2-0.9.6-1.tar.bz2
Adding in variants from /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpplj17zw7/info/recipe/conda_build_config.yaml
INFO:conda_build.variants:Adding in variants from /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpplj17zw7/info/recipe/conda_build_config.yaml
Renaming work directory,  /usr/local/miniconda/conda-bld/pysdl2_1556190636397/work  to  /usr/local/miniconda/conda-bld/pysdl2_1556190636397/work_moved_pysdl2-0.9.6-1_osx-64
Collecting package metadata: ...working... done
Solving environment: ...working... done

## Package Plan ##

  environment location: /usr/local/miniconda/conda-bld/pysdl2_1556190636397/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla


The following NEW packages will be INSTALLED:

    bzip2:           1.0.6-h1de35cc_1002  conda-forge
    ca-certificates: 2019.3.9-hecc5488_0  conda-forge
    libcxx:          8.0.0-2              conda-forge
    libcxxabi:       8.0.0-2              conda-forge
    libffi:          3.2.1-h6de7cb9_1006  conda-forge
    ncurses:         6.1-h0a44026_1002    conda-forge
    openssl:         1.1.1b-h01d97ff_2    conda-forge
    pysdl2:          0.9.6-1              local      
    python:          3.7.3-h0d93f26_0     conda-forge
    readline:        7.0-hcfe32e1_1001    conda-forge
    sdl2:            2.0.9-h6de7cb9_0     conda-forge
    sqlite:          3.26.0-h1765d9f_1001 conda-forge
    tk:              8.6.9-ha441bb4_1001  conda-forge
    xz:              5.2.4-h1de35cc_1001  conda-forge
    zlib:            1.2.11-h1de35cc_1004 conda-forge

Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
WARNING: symlink_conda() is deprecated.
export PREFIX=/usr/local/miniconda/conda-bld/pysdl2_1556190636397/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla
export SRC_DIR=/usr/local/miniconda/conda-bld/pysdl2_1556190636397/test_tmp
import: 'sdl2'
Traceback (most recent call last):
  File "/usr/local/miniconda/conda-bld/pysdl2_1556190636397/test_tmp/run_test.py", line 2, in <module>
    import sdl2
ModuleNotFoundError: No module named 'sdl2'
Tests failed for pysdl2-0.9.6-1.tar.bz2 - moving package to /usr/local/miniconda/conda-bld/broken
WARNING:conda_build.build:Tests failed for pysdl2-0.9.6-1.tar.bz2 - moving package to /usr/local/miniconda/conda-bld/broken
TESTS FAILED: pysdl2-0.9.6-1.tar.bz2
```

**Comment by dschreij** [comment]
That being said, the above is the log for Python 2.7 and with the test it tries to install Python 3.7.3, so that's probably why it fails. Is this due to a misconfiguration of my meta.yaml?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -14,19 +14,17 @@ source:
...
     - patch  # [win]
-    - sdl2
+  host:
+    - pip
     - python
-    - setuptools
-
   run:
     - sdl2
-    - python
```
Comment (ocefpaf) [inline recipe/meta.yaml]: This is your problem.

**Comment by ocefpaf** [comment]
> You're right! That was a lapse on my behalf.

You can also delete the LICENSE.txt file from the recipe.

> That being said, the above is the log for Python 2.7 and with the test it tries to install Python 3.7.3, so that's probably why it fails. Is this due to a misconfiguration of my meta.yaml?

Yep. You removed `python` as a run-time dependency and it will try  to run with whatever is available instead.

File: recipe/meta.yaml (hunk 2 above)
Comment (dschreij) [reply recipe/meta.yaml]: So python should be added to run, even though it's also present under `host`. Got it. To be honest I'm a bit confused with what goes where at the moment. Thanks for your help!

**Comment by dschreij** [comment]
Great! Thanks for the help @ocefpaf

File: recipe/meta.yaml (hunk 2 above)
Comment (ocefpaf) [reply recipe/meta.yaml]: `host` what must be on the host machine during build. If you depend on python to run you must add it to `run`. For more information see https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-host-and-run

**Pull request merged**